ios/MLCSwift/Sources/ObjC/LLMChat.mm

```diff
@@ -97,10 +153,97 @@ - (NSString*)runtimeStatsText {
   return [NSString stringWithUTF8String:ret.c_str()];
 }
 
+- (void)processSystemPrompts {
+  process_system_prompts_func_();
+}
+
 - (void)evaluate {
   LOG(INFO) << "Total-mem-budget=" << os_proc_available_memory() / (1 << 20) << "MB";
   llm_chat_->GetFunction("evaluate")();
   LOG(INFO) << "Left-mem-budget=" << os_proc_available_memory() / (1 << 20) << "MB";
 }
 
+- (void)unloadImageMod {
+  image_mod_unload_func_();
+  first_input_after_image = false;
+}
+
+- (void)reloadImageMod:(NSString*)modelLib modelPath:(NSString*)modelPath {
+  std::string lib_prefix = modelLib.UTF8String;
+  std::string model_path = modelPath.UTF8String;
+  std::replace(lib_prefix.begin(), lib_prefix.end(), '-', '_');
+  lib_prefix += '_';
+  Module lib = (*Registry::Get("runtime.SystemLib"))(lib_prefix);
+  image_mod_reload_func_(lib, model_path);
+  first_input_after_image = false;
+}
+
+- (void)resetImageMod {
+  image_mod_reset_func_();
+  first_input_after_image = false;
+}
+
+- (NSString*)runtimeStatsTextImageMod {
+  std::string ret = image_mod_runtime_stats_text_func_();
```

Comment:
This can be combined with `runtimeStatsText`

Reply:
resolved